Sources/Classes/core/MiniApp.swift

```diff
@@ -204,6 +204,17 @@ public class MiniApp: NSObject {
         try? MiniAppSecureStorage.wipeSecureStorage(for: miniAppId)
     }
 
+    /// Method will send the jsonString to MiniApp
+    /// - Parameters:
+    ///   - miniAppId: Mini AppId String value
+    ///   - miniAppVersion: Mini VersionId String value
+    ///   - jsonString: string to be snt to MiniApp
+    public func sendJsonStringToMiniApp(withId miniAppId: String,
```

Comment:
[In Android](https://github.com/rakutentech/android-miniapp/blob/master/miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/view/MiniAppView.kt#L73), we have written `sendJsonToMiniApp`. Probably, we can align the same name for the public interfaces.

Reply:
@GondiTilak , please change the method name from 'sendJsonStringToMiniApp' to 'sendJsonToMiniApp' as per the tech spec. Also, I don't think this method is required in `MiniApp.swift` class as these interfaces is have shared config.

Reply:
Please create interfaces in MiniAppView class like in Android

Reply:
Could you please remove this method?

Reply:
This is now removed in the latest commit.